release/models/bfd/openconfig-bfd.yang

```diff
@@ -508,6 +508,100 @@ module openconfig-bfd {
     uses bfd-session-microbfd-common;
   }
 
+  grouping bfd-interface-session-config {
+    description
+      "Configuration parameters for a BFD session on an interface.";
+
+    leaf local-address {
+      type oc-inet:ip-address;
+      description
+        "The local IP address used by the system for the single-hope BFD session
+        specified.";
+    }
+
+    leaf remote-address {
+      type oc-inet:ip-address;
+      description
+        "The remote IP destination that should be used by the system for
+        the single-hope BFD session specified.";
+    }
+
+    leaf enabled {
+      type boolean;
+      description
+        "When this leaf is set to true then the single-hope BFD session is enabled
+        - if it is set to false, it is administratively disabled.";
+    }
+
+    leaf demand-mode {
+      type boolean;
+      description
+        "When this leaf is set to true then the single-hope BFD session 
+        is operating in demand-mode.";
+    }
+  }
+
+  grouping bfd-interface-session-state {
+    // placeholder
+    description
+      "Operational state parameters relating to the single-hope BFD session 
+      running on the interface.";
+  }
+
+  grouping bfd-interface-sessions-structural {
+    description
+      "Structural grouping for BFD sessions configuration on interface
+      and state parameters.";
+
+    container sessions {
+      description
+        "Parameters relating to BFD sessions associated with the interface.";
```

Comment:
Can we add something here that describes that these are statically configured single-hop sessions? I presume the use case here is where we don't have a protocol that ties to that BFD session (so there's no `enable` leaf). Is there anything implied by this being created?

Reply:
Description of sessions container changed to specify that these are statically configured single-hop sessions.
Also changed the name of enable leaf to admin-down, to be more specific. When set to true, the session will be administratively down.